packages/i18n/src/locales/en/app.yaml

```diff
@@ -341,3 +341,5 @@ darkTheme: Dark Theme
 hax0rTheme: Hax0r Theme
 lgbtqTheme: LGBTQ Theme
 transTheme: Trans Theme
+editError: Issues with YAML
```

Comment:
The `editError` and `editErrorDesc` are not great translation keys.
There's several places in the UI where things can be edited and where that could cause an error.
This, in combination with the fact that this is added to the sort of catch-all `app.yaml` makes me feel we should:
- Move it to a more specific file since it should only ever be loaded on the Edit YAML view
- Give it a more descriptive name